src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -10084,9 +10080,245 @@
+      events:
+          OvenCavityOperationalState:
+              - OperationalError
+              - OperationCompletion
+      event fields:
+          OvenCavityOperationalState:
+              OperationalError:
+                  - errorState
+              OperationCompletion:
+                  - completionErrorCode
+                  - totalOperationalTime
+                  - pausedTime
+      enum values:
+          MicrowaveOvenMode:
+              ModeTag:
+                  # generic mode tags
+                  - Auto
+                  - Quick
+                  - Quiet
+                  - LowNoise
+                  - LowEnergy
+                  - Vacation
+                  - Min
+                  - Max
+                  - Night
+                  - Day
+                  # MicrowaveOvenMode-specific tags
+                  - Normal
+                  - Defrost
+          OvenMode:
+              ModeTag:
+                  # generic mode tags
+                  - Auto
+                  - Quick
+                  - Quiet
+                  - LowNoise
+                  - LowEnergy
+                  - Vacation
+                  - Min
+                  - Max
+                  - Night
+                  - Day
+                  # OvenMode-specific tags
+                  - Bake
+                  - Convection
+                  - Grill
+                  - Roast
+                  - Clean
+                  - ConvectionBake
+                  - ConvectionRoast
+                  - Warming
+                  - Proofing
+                  - Steam
+          OvenCavityOperationalState:
+              ErrorStateEnum:
+                  - NoError
+                  - UnableToStartOrResume
+                  - UnableToCompleteOperation
+                  - CommandInvalidInState
+              OperationalStateEnum:
+                  - Stopped
+                  - Running
+                  - Paused
+                  - Error
+      bitmaps:
+          MicrowaveOvenControl:
+              - Feature
+      bitmap values:
+          MicrowaveOvenControl:
+              Feature:
+                  - PowerAsNumber
+                  # PowerInWatts is provisional because WATTS is provisional in Matter 1.3
+                  # WATTS is provisional in 1.3 because no one has submitted a DUT for it
+                  - PowerNumberLimits
   provisional:
       device types:
           - BatteryStorage
           - HeatPump
           - SolarPower
           - WaterHeater
+      attributes:
+          MicrowaveOvenControl:
+              - SupportedWatts
+              - SelectedWattIndex
+      command fields:
+          MicrowaveOvenControl:
+              SetCookingParameters:
+                  - wattSettingIndex
+      bitmap values:
+          MicrowaveOvenControl:
+              Feature:
+                  - PowerInWatts
+  removed:
+      attributes:
+          OvenMode:
+              - StartUpMode
+              - OnMode
```

Comment:
Please document why these are "removed" even though they are still in the XML?

And please file an issue to get the XML fixed.  Same for OvenCavityOperationalState.